### DISCUSSION TITLE
How to add a transport for the Logger (Pino) on the fly?

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#3349] [Help]
Hi

I read the logger documentation https://docs.adonisjs.com/guides/logger#using-pino-transports how to change the transport of the adonis logger.
Buy how can a add! a second transport from a Serviceprovider? My wish is to have the "normal" logging and add a second transport via Serviceprovider.

Bye!

**Comment by josh231101** [comment]
Any updates on this??
